client/src/App.test.js

```diff
@@ -14,11 +14,6 @@ const renderComponent = () =>
         </FetchMock>
     );
 
-test('verifies fetch is called', async () => {
```

Comment:
Don't forget to clean up any unused imports like "wait" from line 3 and renderComponent from line 6